### PR TITLE
[SRVKS-1171] [RELEASE-1.12] Graceful shutdown

### PR DESCRIPTION
**Pull request by skonto**
- Backport of https://github.com/knative-extensions/net-kourier/pull/1203

### CODE REVIEW

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

File: openshift/release/artifacts/net-kourier.yaml [hunk 1]
```diff
@@ -526,7 +542,9 @@ spec:
               memory: 200Mi
             limits:
               cpu: "1"
-              memory: 500Mi
+              memory: 800Mi
```
Comment (skonto) [inline openshift/release/artifacts/net-kourier.yaml:545]: Bringing this from the upstream. @ReToCode do we need to update any docs if we bring this in?

Comment (ReToCode) [reply openshift/release/artifacts/net-kourier.yaml:545]: I don't think so, we have not documented limits, just what is a typical use with a certain amount of service. Maybe as a release note to raise awareness?

**Comment by skonto** [comment]
/unhold

Tests passed.

**Pull request merged**